Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM debian:jessie
+MAINTAINER Julian Schlichtholz <julian.schlichtholz@gmail.com>
+
+# Install dependencies.
+RUN apt-get update
+RUN apt-get install -y python3
+RUN apt-get install -y python3-pip
+RUN apt-get install -y python3-numpy
+RUN apt-get install -y python3-scipy
+RUN apt-get install -y python3-pandas
+RUN apt-get install -y python3-nose
+RUN apt-get install -y python3-patsy
+RUN apt-get install -y python3-msgpack
+RUN apt-get install -y python3-redis
+RUN apt-get install -y python3-flask
+RUN apt-get install -y python3-simplejson
+RUN apt-get install -y python3-mock
+RUN apt-get install -y python3-pep8
+RUN rm -rf /var/lib/apt/lists/*
```

Comment:
i know this PR is defunct, but just thought I'd offer some tips re:Dockerfiles.

chaining related RUN commands with `&&` will reduce the layer count, size, and build-time of your resulting docker image. "Related" being a set of commands that should happen together in the build process and the results are invalidated if one of the individual parts is changed; as well as any setup/cleanup commands that go with them. So in this case, the `apt` lines are related. `apt-get update` is the setup, `rm -rf /var/lib/apt/lists/*` is the cleanup, and the `apt-get install` lines are the "work" being done.

Here's a good write-up with an example that also illustrates some other concepts: http://tech.paulcz.net/2015/03/optimizing-your-dockerfiles/

Also, in general you should use `COPY` instead of `ADD`: http://www.centurylinklabs.com/dockerfile-add-vs-copy/

Hope that helps!